src/ddmd/dsymbolsem.d

```diff
@@ -5144,6 +5144,11 @@ extern(C++) final class DsymbolSemanticVisitor : Visitor
 
     override void visit(UnitTestDeclaration utd)
     {
+        // the identifier has to be generated here in order to be able to link
+        // or not the files are compiled separately or all at once.
```

Comment:
I can guess what the sentence means, but it is just a guess. Please fix grammar!

Reply:
I think that was a rebase issue. Fixing it all the same.